messaging-experiments.yaml

```diff
@@ -467,3 +467,306 @@
+              icon: data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABmJLR0QAAAAAAAD5Q7t/AAAACXBIWXMAAAsSAAALEgHS3X78AAACCUlEQVRo3u2YPU8UURSGn10nikCCUSJaaGNiZWWjnf6BN6FCLIiGQhtNMBpDZWc0BgoV0WCBECoozy9Q/4C1iQU2QjDKRwS2EYvdTTaTYWC5e+9k8T7d3jvnPefZ3dmdmRKApCHgNtCDGxvAjJl9aLZQUj+wACS1pW/AJTPbzqtLJN0DXjsO3sh1SafMbLzJuisNwwNcAE4D3/OKysDjFg5fZ9RD5q4C5zzk9krqDiXgiyPtLhCEKFA0iXvE/pDUBfTlHHIiY+28pLwZfwQRkNRJ9Y+pr8nSz3vsfw31Feo9wPD74WLbnwOhBLaAHQ+5lSDngJmtSBoELuccdg24mlp7C6zn1HwM9itkZvPA/G77kp5lCDw3sz0v5tqaKFA0h0Lgl4fcP2a2FkpgxkPudIjh6wKjwDit+SR+A6+Ahweo/QRsNrz+AiyFeiMi/y2l0A1r9wbHHGM2zawCUJJ0FBgDhoEu12BgDhgxs63U4N3AFDCA+xOLbeAd8CgBXgD3HQPrdAJ3aiIPUntjwM0W9ekARoCfZWCoRaGN3M1Yu+Whz40ycNJD8HFJ6QfFHR769ByKa6G2JgoUTRQomihQNFGgaKJA0USBovEp8DeUwLKH3DUz20itrXros1wG3ngInsxY89FnIgGeAhWqt3xnHQOXgPfAy4y9J8AK1dvNMw49doBFYMLMZv8BtVRph4qoHLkAAAAASUVORK5CYII=
+              rating: 4.6
+              users: 226263
+              author: Mozilla Firefox
+              amo_url: https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/
+            text: Organize your online life like the rest of your life! Use Firefox containers to keep your shopping, social, and work habits separate.
+            buttons:
+              primary:
+                label:
+                  value: Add now
+                  attributes:
+                    accesskey: A
+                action:
+                  type: INSTALL_ADDON_FROM_URL
+                  data:
+                    url: null
+                    telemetrySource: amo
+              secondary:
+                - label:
+                    string_id: cfr-doorhanger-extension-cancel-button
+                  action:
+                    type: CANCEL
+                - label:
+                    string_id: cfr-doorhanger-extension-never-show-recommendation
+                - label:
+                    string_id: cfr-doorhanger-extension-manage-settings-button
+                  action:
+                    type: OPEN_PREFERENCES_PAGE
+                    data:
+                      category: general-cfraddons
+          frequency:
+            lifetime: 3
+          targeting: (addonsInfo.addons|keys intersect ["@testpilot-containers"])|length == 0 && ((currentDate|date - profileAgeCreated) / 604800000) >= 1 && (topFrecentSites[.frecency > 100]|mapToProperty("host") intersect ["www.youtube.com","www.facebook.com","www.twitter.com","www.instagram.com","www.reddit.com","www.pinterest.com","www.linkedin.com","www.whatsapp.com","www.tiktok.com","www.messenger.com","www.tumblr.com","www.groupme.com","www.flickr.com","youtube.com","facebook.com","twitter.com","instagram.com","reddit.com","pinterest.com","linkedin.com","whatsapp.com","tiktok.com","messenger.com","tumblr.com","groupme.com","flickr.com"])|length
+          trigger:
+            id: openURL
+            params:
```

Comment:
Per experiment design, I believe the trigger of this branch should be `navigating to social meida sites`

Reply:
The targeting is `Targeting users who visited shopping sites, i.e. shopping sites in the topFrecentSites list.`